sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -57,16 +61,14 @@
 class GenBank_SBOL3_Converter:
     gb2so_map = {}
     so2gb_map = {}
-    default_SO_ontology = "SO:0000110"
-    default_GB_ontology = ""  # TODO: Whats the default here?
+    DEFAULT_SO_TERM = "SO:0000110"
+    DEFAULT_GB_TERM = "todo"  # TODO: Whats the default here?
```

Comment:
```suggestion
    DEFAULT_GB_TERM = "misc_feature"
```
Per a conversation with @cjmyers today